docs/distributions/ubuntu/upgrade.md

```diff
@@ -0,0 +1,55 @@
+# Upgrade
+
+Recommendations:
+
+* Perform upgrade from LTS to LTS.
+
+Warnings:
+
+* If you have important data and workspace, make sure that you have backup.
+* Make sure you have enough disk space.
+* **Ensure that you met all the below requirements so that you can recover in case of failed upgrade.**
+
+# Hardware Requirements
+
+* USB-C to Ethernet adapter **OR** Smartphone capable of USB tethering to provide internet in case of emergency
+* A cable that can connect one of the above objects to your Mac
+
+# Kernel Requirements
+
+Check the latest official Ubuntu release and find its kernel version. You need to install the same or higher custom kernel version for your current Ubuntu.
+
+# Install Procedure
+
+Most probably you are using a LTS version of the Ubuntu right now. If a newer LTS Ubuntu is released in April, you will not be notified until Canonical release the first point release (e.g. 22.04.1) which normally out around August. However, you can still upgrade to the final version before August.
+
+1. Open a terminal and:
+
+    1. If it's after first point release, do `sudo do-release-upgrade`.
```

Comment:
Probably add the we can use the software updater here as well

Reply:
Oh, I forgot to mention that GUI. I will add.